man/paru.8

```diff
@@ -234,19 +233,21 @@ it is up to the user what upgrades they skip.
 Do not show the upgrade menu.
 
 .TP
-.B \-\-removemake [yes|no|ask]
```

Comment:
`<...>` Means required, while `[...]` means optional. `--removemake` defaults to yes in the sense that `--removemake` = `--removemake=yes`, to get ask you do `--removemake=ask.